### DISCUSSION TITLE
Start button brings up wrong start menu after Windows update

### DISCUSSION THREAD

**Discussion by weishen1121** [valinet/ExplorerPatcher#1544] [Q&A] [answered]
After the most recent windows 11 update, whenever I hit the Start button on my keyboard, it brings up the Windows 11 style start menu. If I click on my start button with my mouse, then it works properly and brings up Classic Start Menu.

I tried re-installing explorerpatcher and classic start menu but no luck. 

Any ideas??

**Comment by pyrates999** [comment]
Please post your windows build number by running winver.

**Reply by weishen1121** [reply]
> winver

Version 22H2 - OS Build 22621.1778

**Reply by Aadii8430** [reply]
happening with mw also, from the last two days after the window 11 update, the start menu and Search menu not opening, when I click search or start loading start and then with window comes and then disappears but the start and search not opened

**Reply by pyrates999** [reply]
> happening with mw also, from the last two days after the window 11 update, the start menu and Search menu not opening, when I click search or start loading start and then with window comes and then disappears but the start and search not opened

You're also running the same preview update.  Uninstall it and don't install preview updates next time.

**Reply by Aadii8430** [reply]
Already uninstall 2 days ago, with that update i face many more issues also , that's why i instantly uninstall it that time, and now i chek again its uninstalled but still the issue exist, start and search not opening

**Comment by pyrates999** [comment, marked as answer]
That's a preview update you got installed: https://support.microsoft.com/en-us/topic/may-24-2023-kb5026446-os-build-22621-1778-preview-3c547100-7a73-4ae6-bb7d-ebd02e87dc04

Uninstall it and reboot and see if it works again.

Microsoft does not provide symbols to download for preview updates and insider builds: https://github.com/valinet/ExplorerPatcher/wiki/Symbols

**Comment by pyrates999** [comment]
Then do the following:

1. uninstall explorer patcher
2. delete the registry key: "Computer\HKEY_CURRENT_USER\Software\ExplorerPatcher"
3. reboot
4. install explorer patcher
5. verify you get a notification that symbols have been downloaded
6. restart windows explorer using either the task manager or in the properties window of explorer patcher

Then apply your settings again in explorer patcher to see if it fixes it.

**Reply by mgrant723** [reply]
this resolved it for me... fwiw I do not have preview/insider builds for windows installed on this machine so this may impact  other users but these steps do resolve it.
build is 10.0.22621.1848

**Reply by Aadii8430** [reply]
i do same but still same problem, and no  5 step not happened after again installation. after installation can,t get  notification that symbols have been downloaded

**Reply by pyrates999** [reply]
Do you have any third party firewall software installed?  You might need to deactivate it temporarily because it might be preventing the symbols from being downloaded.

**Reply by Aadii8430** [reply]
Yes i have bitdefender total security, but i already tried that, Disabled all security and firewell etc but still don't know why still problem exist, anywat to manually install that symbols?

**Reply by pyrates999** [reply]
What version and build of windows do you have installed?  You can find out by running winver.

**Reply by Aadii8430** [reply]
window 11 home , Version  22H2, OS Biuld 22621.1413

**Reply by pyrates999** [reply]
I would install the June update then: https://support.microsoft.com/en-us/topic/june-13-2023-kb5027231-os-build-22621-1848-8f903600-1293-4431-9c6b-736a4049666c

It's not a preview update, so symbols are available to download for it.

Before installing, make sure explorer patcher is not installed and that you deleted the registry key: "Computer\HKEY_CURRENT_USER\Software\ExplorerPatcher"

**Reply by Aadii8430** [reply]
Already done that, first it was little bit working, just start and search option was not working, but now when i completely uninstall it and also delete its registry folder completely, so now when i install it its totally not working, anywhere on the taskbar when i try to click , cursor star loadinh them white screen for 2 second and then finish , and after 3 to 4 tries like that, its completely stop working and i got error, screenshot attached . Also after installation i am not getting any notification regarding symbol download , even though all firewall etc are disabled 😕
![Uploading IMG20230618190941.jpg…]()

**Reply by pyrates999** [reply]
The Bit defender firewall is blocking the symbols from being downloaded.  I also can't see the image you uploaded.

**Reply by Aadii8430** [reply]
Bitdefender is already disabled , image is here 
![IMG20230618190941](https://github.com/valinet/ExplorerPatcher/assets/63859896/05fb3989-27ed-4b6f-8f07-742d50d5aa91)

**Reply by pyrates999** [reply]
Did you check notifications that the symbols were downloaded?

https://github.com/valinet/ExplorerPatcher/wiki/Symbols

**Reply by Aadii8430** [reply]
Not downloaded

**Reply by pyrates999** [reply]
Something is blocking the symbols from being downloaded.  Explorer patcher won't work without the symbols being downloaded.

Do you have any other firewalls installed beside bit defender?

**Comment by vippium** [comment]
There is no issue with me regarding preview updates, even if I use Canary channel too, there is no issue I faced like this with EP... I think you guys are doing something wrong.

**Reply by pyrates999** [reply]
https://github.com/valinet/ExplorerPatcher/wiki/Symbols

**Reply by Aadii8430** [reply]
I don't do anything wrong sir, and now whole taskbar not working with me, after installing EP preview symbols not auto downloading

**Comment by Aadii8430** [comment]
No , but the latest june window update releasee not working with me , those
release not release , as you said last time that june update support
symbols download, but my window june update not doing well, and except bit
defender I don't have any other firmware installed

**Reply by pyrates999** [reply]
Ok, disable the bitdefender firewall in bitdefender.  Then open up settings and search for firewall.  Check in Firewall & Network Protection.

I'm using windows 10, so it might be a bit different.  You need to check that windows itself shows that the firewall is disabled.

Here's someone else that got it working by disabling their third party firewall: https://github.com/valinet/ExplorerPatcher/issues/1548#issuecomment-1601430443

**Comment by Aadii8430** [comment]
Ok, I'll check it out.
